src/frontend/src/lib/components/tokens/TokensDisplayHandler.svelte

```diff
@@ -1,55 +1,18 @@
 <script lang="ts">
-	import type { Token, TokenUi } from '$lib/types/token';
-	import { pointerEventStore } from '$lib/stores/events.store';
+	import type { Token } from '$lib/types/token';
 	import { pointerEventsHandler } from '$lib/utils/events.utils';
-	import { debounce } from '@dfinity/utils';
 	import { hideZeroBalancesStore } from '$lib/stores/settings.store';
-	import { combinedDerivedSortedNetworkTokensUi } from '$lib/derived/network-tokens.derived';
+	import { combinedDerivedTokensToDisplay } from '$lib/derived/tokens-to-display.derived';
```

Comment:
`combinedDerivedTokensUi`? Just an idea but given we have `TokenUi` maybe it makes sense?

Reply:
yes, better!

in a separate PR after this, i will move all these derived UI to a separate module, to concentrate them